client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -41,6 +45,7 @@ class PacmanGame extends Component {
     moveGhostsCount: 0,
     scatterGhostspath: [],
     scatterStart: 75,
+    playerId: uuid.v1(),
```

Comment:
Looking from the clients perspective, this is `myId` rather than just any `playerId`. Use a better variable name.